api/emoji.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
```

Comment:
Can you revert the changes to the copyright lines in the 3 places you changed them? We usually just leave that year as when the file was originally created, and this is also definitely conflicting with the changes I made to them a couple days ago